kicker/app/connectors/argo/models/ArgoWorkflow.py

```diff
@@ -43,19 +43,22 @@ def _create_manifest(request: KickerMessage, params: Dict[str, str]) -> Dict[str
 def _get_container_tag_params_for_given_workflow(workflow_name: str, tag_params: ContainerTagParams) -> Dict[str, str]:
     params = {}
     if workflow_name == WorkflowType.VERIFY_OVERFORINGSPAKKE:
-        params = {**tag_params.checksum,
-                  **tag_params.avscan,
-                  **tag_params.mailer,
-                  **tag_params.unpack,
-                  **tag_params.arkade5,
-                  **tag_params.artifact_writer}
+        params = {
+            **tag_params.checksum,
+            **tag_params.avscan,
+            **tag_params.mailer,
+            **tag_params.unpack,
+            **tag_params.arkade5,
+            **tag_params.artifact_writer,
+        }
     elif workflow_name == WorkflowType.NOTIFY_AVGIVER:
         params = {**tag_params.mailer}
```

Comment:
```suggestion
        params = {
            **tag_params.mailer,
        }
```
Så får vi det likt mellom de tre :smile: